pkg/e2e/fixtures.go

```diff
@@ -545,3 +593,31 @@ func EnsureServiceAttachment(s *Sandbox, saName, svcName, subnetName string) (*s
 func DeleteServiceAttachment(s *Sandbox, saName string) error {
 	return s.f.SAClient.Delete(s.Namespace, saName)
 }
+
+// GetL4LBForSvcParams extract GCP resources name from NetLB service annotations
+func GetL4LBForSvcParams(s *Sandbox, svcName string) (*fuzz.L4LBForSvcParams, error) {
```

Comment:
this should be in helpers rather than fixtures

Reply:
done